lint-tasks.yml

```diff
@@ -67,6 +67,7 @@ tasks:
       - "{{.G_WEBUI_SRC_DIR}}/server/**/*.jsx"
       - "{{.G_WEBUI_SRC_DIR}}/tests/**/*.js"
       - "{{.G_WEBUI_SRC_DIR}}/tests/**/*.jsx"
+      - "{{.ROOT_DIR}}/deps-tasks.yml"
```

Comment:
Why is this in js-check's sources?

Reply:
pretty sure I misunderstood something when adding this line